versions-maven-plugin/src/it/it-unlock-snapshots-issue-134/pom.xml

```diff
@@ -0,0 +1,22 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <groupId>localhost</groupId>
+  <artifactId>it-use-releases-issue-134</artifactId>
+  <version>1.0</version>
+  <packaging>pom</packaging>
+  <name>Check if boms are upated</name>
+
+  <dependencyManagement>
+    <dependencies>
+      <dependency>
+        <groupId>org.codehaus.plexus</groupId>
+        <artifactId>plexus-utils</artifactId>
+        <version>1.5.1-20090311.051742-1</version>
```

Comment:
Even if today such version of snapshot exists - can be removed, why not use some of dumy dependencies?

Reply:
I coundt reproduce it with dummy dependencies. I did use dummy dependencies wherever I could.

Reply:
As an alternative or a temporary workaround, I can remove these tree integration tests using non-dummy artifacts. Should I?

Reply:
Fixed